metecho/api/sf_run_flow.py

```diff
@@ -226,10 +227,15 @@ def get_access_token(*, org_result, scratch_org_config):
     the scratch org is created. This must be completed once in order for future
     access tokens to be obtained using the JWT token flow.
     """
-    oauth = SalesforceOAuth2(
-        SF_CLIENT_ID, SF_CLIENT_SECRET, SF_CALLBACK_URL, scratch_org_config.instance_url
+    oauth_config = OAuth2ClientConfig(
+        client_id=SF_CLIENT_ID,
+        client_secret=SF_CLIENT_SECRET,
+        redirect_uri=SF_CALLBACK_URL,
+        auth_uri=f"{scratch_org_config.instance_url}/services/oauth2/authorize",
+        token_uri=f"{scratch_org_config.instance_url}/services/oauth2/token",
     )
-    auth_result = oauth.get_token(org_result["AuthCode"]).json()
+    oauth = OAuth2Client(oauth_config)
+    auth_result = oauth.auth_code_grant(org_result["AuthCode"]).json()
```

Comment:
@davisagli This was a rough first attempt to upgrade CCI based on changes in https://github.com/SFDO-Tooling/CumulusCI/pull/2602. It seems to work locally, but you're in a better place to review/improve this.

Reply:
@jgerigmeyer we should probably also explicitly specify what oauth scopes we're requesting, like here: https://github.com/SFDO-Tooling/MetaDeploy/blob/ee9d826ea4e2ae63c744aa7708acfaca94f252a3/metadeploy/api/salesforce.py#L218

But otherwise looks right.

Reply:
Thanks! (I guess I should've looked for that MD PR earlier! 😂)